packages/devtools/devtools/src/panels/client/LoggingPanel/LoggingPanel.tsx

```diff
@@ -100,7 +100,7 @@ export const LoggingPanel = () => {
       { value: 'warn', label: 'Warn' },
       { value: 'error', label: 'Error' },
 
-      // TOOD(burdon): Factor out.
+      // TOOD(burdon): Factor out. Move to separate pull down.
```

Comment:
I can understand wanting to delineate these things in some way bu why a separate pull down? grouping seems like it would be sufficient to me

Reply:
right. not a priority, but ability for user to define levels.